compose/cli/docker_client.py

```diff
@@ -14,7 +15,27 @@
 log = logging.getLogger(__name__)
 
 
-def docker_client(version=None):
+def tls_config_from_options(options):
+    tls = options.get('--tls', False)
+    ca_cert = options.get('--tlscacert')
+    cert = options.get('--tlscert')
+    key = options.get('--tlskey')
+    verify = options.get('--tlsverify')
+
+    if tls is True:
+        return True
```

Comment:
I thought so, but checking again, it seems another valid option is `docker --tls --tlscert ... --tlskey ...`  which means "Authenticate with client certificate, do not authenticate server based on given CA" (what we would do here if `--tls` was omitted). I'll update to match.